### PR TITLE
More contacts info; now serves miscuser xml file too

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: converters/download [hunk 1]
```diff
@@ -3,40 +3,50 @@
...
+            euid = os.geteuid()
+            proxypath = "/tmp/x509up_u%d" % euid
+            if euid == 0:
+                cert = '/etc/grid-security/hostcert.pem'
+                key = '/etc/grid-security/hostkey.pem'
+            elif os.path.exists(proxypath):
+                cert = key = proxypath
+            else:
+                cert = os.path.expanduser("~/.globus/usercert.pem")
+                key = os.path.expanduser("~/.globus/userkey.pem")
```
Comment (brianhlin) [inline converters/download]: Can we add a TODO to respect `X509_USER_CERT` and `X509_USER_KEY` env vars?

File: webapp/contacts_reader.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        return self.yaml_data["ContactInformation"].get("PrimaryPhone", None)
+
+    @property
+    def sms_address(self):
+        return self.yaml_data["ContactInformation"].get("SMSAddress", None)
+
+    @property
+    def dns(self):
+        dns = self.yaml_data["ContactInformation"].get("DNs", None)
+        return dns
```
Comment (brianhlin) [inline webapp/contacts_reader.py]: This is a nitpick but I was at first confused why we had contact DNS information

File: converters/download (hunk 1 above)
Comment (matyasselmeci) [reply converters/download]: That's quick enough to fix. Updated.

File: webapp/contacts_reader.py (hunk 2 above)
Comment (matyasselmeci) [reply webapp/contacts_reader.py]: Yeah, them's the breaks for language naming conventions...

**Pull request merged**